src/olcs/FeatureConverter.js

```diff
@@ -213,7 +213,7 @@ exports.prototype.wrapFillAndOutlineGeometries = function(layer, feature, olGeom
     primitives.add(p1);
   }
 
-  if (olStyle.getStroke()) {
+  if (olStyle.getStroke() && outlineGeometry) {
```

Comment:
I added this extra check because the code that creates the polylines for polygon outlines never sets outlineGeometry. This is another bit of code that can be removed when Cesium adds support for PolygonOutlines on terrain natively.